code/modules/trader/trade_window.dm

```diff
@@ -321,7 +323,8 @@
 /obj/structure/trade_window/say(var/message)
 	if(!trader_language)
 		trader_language = all_languages[LANGUAGE_VOX]
-	..(message, trader_language)
+	if(world.time>time_last_speech+2 SECONDS)
```

Comment:
this isn't how you should do this, because sometimes the window actually does try to send two different messages one after another right away; if the problem is with people clicking on it, add the delay to attack_hand